mutt/ctype2.h

```diff
@@ -0,0 +1,35 @@
+/**
+ * @file
+ * ctype(3) wrapper functions
```

Comment:
Mind explaining the reason from wrapping a bit here?

Reply:
Done.